example/tls_client.cpp

```diff
@@ -39,7 +39,14 @@ int main(int argc, char** argv) {
     // Setup client
     c->set_client_id("cid1");
     c->set_clean_session(true);
+
+#if defined(MQTT_USE_TLS)
+#if defined(MQTT_USE_GNU_TLS)
+    c->get_ssl_context().use_verify_file(cacert, tls::context::pem);
```

Comment:
Why is the interface for this different?

Reply:
The boost-asio-gnu-tls repository exposes APIs that begin with `use_` and that's why I added a new API to it called `use_verify_file` to be consistent with the other APIs.

Reply:
Why are they different though? That's a major pain point for projects that consume the GnuTLS varient of the API.

Reply:
Yeah, makes sense. Let me follow up there and try getting the name changed to be the same as with boost/asio/ssl.hpp

Reply:
I've sent a pull-request to update the API name and also switched to using `load_verify_file` here. Resolving the comment